src/index.ts

```diff
@@ -76,54 +120,103 @@ export type ImgixUrlQueryParams = {
     faceindex?: number;
     facepad?: number;
     'min-h'?: number;
+    'mark-w'?: number;
+    'mark-align'?: ImgixMarkAlign;
+    'mark-pad'?: number;
+    markY?: number;
```

Comment:
Shouldn't this be `mark-y`? Do they support both notations?

Reply:
good catch! I think they do support it but it's not documented